imgui-render-vulkan/imgui/buildfile

```diff
@@ -0,0 +1,64 @@
+intf_libs = # Interface dependencies.
+import intf_libs += imgui%lib{imgui}
+impl_libs = # Implementation dependencies.
+
+platform_libs =
+
+switch $cxx.target.class
+{
+    case 'windows'
```

Comment:
I don't think you need to do anything other than import https://cppget.org/Vulkan-Headers?q=vulkan or https://cppget.org/Vulkan-Hpp?q=vulkan as it will do the work for you, or at least it did when I tried in the past.

Reply:
using the `vulkan-meta` packages solves this.